### ISSUE TITLE
Allow next throw only after every client reported that "STOOOOP" has finished on their side

### ISSUE THREAD

**Issue by JustMyGithub** [ApolloLV/flunkysimulator#54]

**Comment by ApolloLV** [comment]
This might become possible to do when we improve the server-side stream handling in a future version. Right now the server does not know how many javascript clients are connected. So even if the clients told the server they are finished playing, he would not be able to figure out whether that was everyone. For now, we will have to live with the hard-coded cool down period of #37.

**Comment by JustMyGithub** [comment]
How is that possible? Clients are provided with status update (next through etc.), so the server needs to know the number of clients somehow ?!

**Comment by ApolloLV** [comment]
No, the server just has a list of streams and it pours new game states into every one of them. He does not keep track of which username is associated with each stream, or even whether the user behind a stream is registered or not.

**Comment by stheid** [comment]
related to #27
